CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # osquery Changelog
 
+
+<a name="5.2.1"></a>
+## [5.2.1](https://github.com/osquery/osquery/releases/tag/5.2.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.1.0...5.2.1)
+
+Osquery 5.2.1 brings native Apple Silicon (M1) support to the macOS
+platform. It also represents a comprehensive review and update of our
+third-party dependancies. To support this work, the developer docs
+have been updated, as have several parts of the build system
+
+This release represents commits from 9 contributors! Thank you all.
+
+### New Features
+
+- Apple Silicon support ([#7330](https://github.com/osquery/osquery/pull/7330))
+
+### Table Changes
+
+- Update `time` table to reflect UTC values ([#7276](https://github.com/osquery/osquery/pull/7276))
+- Hide the deprecated `antispyware` column in `windows_security_center` ([#7411](https://github.com/osquery/osquery/pull/7411))
+
+### Bug Fixes
+
```

Comment:
Either here on under a different heading, we updated two third-party libraries with recently disclosed CVEs.

 - libs: Update libarchive to version 3.5.2
 - libs: Update libmagic/file to version 5.40